core/server/master/src/main/java/alluxio/master/file/loadmanager/ExecutionWorkerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.loadmanager;
+
+import alluxio.grpc.LoadResponse;
+import alluxio.master.file.loadmanager.load.BlockBatch;
+
+/**
+ * Class for assigned worker info.
+ */
+public class ExecutionWorkerInfo {
+  /**
+   * Execute a task.
+   * @param blockBatch blockBatch
+   * @return LoadResponse
+   */
+  public LoadResponse execute(BlockBatch blockBatch) {
```

Comment:
feel like better to keep the abstraction in sync, ex. loadResponse vs load request, block statuses with blockBatch

Reply:
Yeah, good point, will make all the related changes in the next PR.